diff --git a/libmamba/CMakeLists.txt b/libmamba/CMakeLists.txt
index b85d26a..02593c0 100644
--- a/libmamba/CMakeLists.txt
+++ b/libmamba/CMakeLists.txt
@@ -262,6 +262,8 @@ macro(libmamba_create_target target_name linkage deps_linkage output_name)
 
     if (${deps_linkage_upper} STREQUAL "STATIC")
         message("   -> Statically linking against libmamba (static) dependencies")
+        find_package(spdlog CONFIG REQUIRED)
+        find_package(CLI11 CONFIG REQUIRED)
         if (UNIX)
 
             set(REQUIRED_STATIC_DEPS
@@ -327,7 +329,12 @@ macro(libmamba_create_target target_name linkage deps_linkage output_name)
                 target_link_options(libmamba-full-static PUBLIC -static-libstdc++ -static-libgcc)
             endif()
 
-            target_link_libraries(libmamba-full-static PUBLIC ${STATIC_DEPS} ${MAMBA_FORCE_DYNAMIC_LIBS})
+            target_link_libraries(libmamba-full-static PUBLIC
+                ${STATIC_DEPS}
+                ${MAMBA_FORCE_DYNAMIC_LIBS}
+                spdlog::spdlog
+                CLI11::CLI11
+                )
 
         elseif (WIN32)
 
@@ -355,6 +362,9 @@ macro(libmamba_create_target target_name linkage deps_linkage output_name)
             find_library(CHARSET_LIBRARY NAMES libcharset)
             message("Found: ${LIBXML2_LIBRARY} ${ICONV_LIBRARY} ${CHARSET_LIBRARY}")
 
+            target_link_libraries(${target_name} PRIVATE
+                spdlog::spdlog
+                CLI11::CLI11)
             target_link_libraries(${target_name} PUBLIC
                 ${CRYPTO_LIBRARIES}
                 ${LibArchive_LIBRARY}
@@ -393,6 +403,8 @@ macro(libmamba_create_target target_name linkage deps_linkage output_name)
         find_package(yaml-cpp CONFIG REQUIRED)
         find_package(reproc++ CONFIG REQUIRED)
         find_package(tl-expected REQUIRED)
+        find_package(spdlog CONFIG REQUIRED)
+        find_package(CLI11 CONFIG REQUIRED)
 
         set(LIBMAMBA_LIBRARIES_DEPS
             ${LIBSOLV_LIBRARIES}
