diff --git a/libmamba/include/mamba/api/configuration_impl.hpp b/libmamba/include/mamba/api/configuration_impl.hpp
index a8e6e72..819603e 100644
--- a/libmamba/include/mamba/api/configuration_impl.hpp
+++ b/libmamba/include/mamba/api/configuration_impl.hpp
@@ -11,6 +11,12 @@
 #include "mamba/core/context.hpp"
 #include "mamba/core/mamba_fs.hpp"
 
+namespace {
+namespace local_detail {
+spdlog::string_view_t level_string_views[] SPDLOG_LEVEL_NAMES;
+}
+}
+
 namespace mamba
 {
     namespace detail
@@ -364,10 +370,10 @@ namespace YAML
             using namespace spdlog::level;
 
             auto name = node.as<std::string>();
-            auto it = std::find(std::begin(level_string_views), std::end(level_string_views), name);
-            if (it != std::end(level_string_views))
+            auto it = std::find(std::begin(local_detail::level_string_views), std::end(local_detail::level_string_views), name);
+            if (it != std::end(local_detail::level_string_views))
             {
-                rhs = static_cast<level_enum>(it - std::begin(level_string_views));
+                rhs = static_cast<level_enum>(it - std::begin(local_detail::level_string_views));
                 return true;
             }
 
