diff --git a/libmamba/include/mamba/core/progress_bar.hpp b/libmamba/include/mamba/core/progress_bar.hpp
index 91aaf84..0fc7956 100644
--- a/libmamba/include/mamba/core/progress_bar.hpp
+++ b/libmamba/include/mamba/core/progress_bar.hpp
@@ -8,6 +8,7 @@
 #define MAMBA_CORE_PROGRESS_BAR_HPP
 
 #include "spdlog/fmt/fmt.h"
+#define FMT_FUNC inline
 #include "spdlog/fmt/bundled/color.h"
 
 #include <iomanip>
